.azure-pipelines/pull_request.yml

```diff
@@ -64,12 +78,60 @@ stages:
         steps:
           - checkout: self
             submodules: true
-          - powershell: Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
-            displayName: Add Conda to PATH
+          - powershell: |
+              Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
+              Write-Host "##vso[task.setvariable variable=sauce]$CONDA"
+            displayName: Configure Conda directories
           - script: |
               conda --version
               conda install --yes conda-build
             displayName: 'Conda configuration'
           - script: |
               conda build -c conda-forge ./conda
             displayName: 'Package build'
+          - task: PublishBuildArtifacts@1
+            inputs:
+              PathtoPublish: '$(conda_dir)/conda-bld/win-64'
+              ArtifactName: 'win-64'
+            displayName: 'Publish Conda package artefacts'
+
+  - stage: 'post_build_checks'
+    displayName: 'Post Build Checks'
+
+    jobs:
+      - job: 'documentation'
+        displayName: 'documentation'
+        pool:
+          vmImage: 'ubuntu-18.04'
+        variables:
+          packages_dir: '$(Build.StagingDirectory)/packages'
+          docs_dir: '$(Build.StagingDirectory)/docs'
+        steps:
+          - bash: |
+              set -ex
+              mkdir -p "$(packages_dir)"
+              mkdir -p "$(docs_dir)"
+            displayName: 'Make directories'
+          - task: DownloadBuildArtifacts@0
+            inputs:
+              buildType: 'current'
+              specificBuildWithTriggering: true
+              downloadPath: '$(packages_dir)'
+          - bash: echo "##vso[task.prependpath]$CONDA/bin"
+            displayName: Add Conda to PATH
+          - bash: conda create -n scipp_docs -f environment.yml
+            displayName: 'Create Conda environment'
+          - bash: conda activate scipp_docs
+            displayName: 'Activate Conda environment'
+          - bash: conda install "$(packages_dir)/"scipp-*.tar.bz2
+            displayName: 'Install Scipp package'
+          - bash: |
+              set -ex
+              cd docs
+              sphinx-build . "$(docs_dir)"
+            displayName: 'Build documentation'
+          - task: PublishBuildArtifacts@1
+            inputs:
+              PathtoPublish: '$(docs_dir)'
```

Comment:
Is this where it pushes to readthedocs?

Reply:
If so, why is it in in `pull_request.yml`?

Reply:
No, this stores a copy in Azure artifacts or inspection.